source/sgp_mode/Tasks.h

```diff
@@ -117,6 +124,28 @@ class TaskSet {
       n_succeeds_sym.push_back(emp::NewPtr<std::atomic<size_t>>(0));
     }
   }
+  /**
+   * Input: A pointer to an organism's square frequency map, as well as the value of the current output
```

Comment:
This should have information about all the inputs, so also the task and the cpustate

Reply:
Do you actually use task or state? If not, you should just remove them